bin/all_sky_search/pycbc_template_kde_max

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+import numpy, h5py, argparse, logging
+from pycbc import init_logging
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument('--template-files', nargs='+', required=True,
```

Comment:
I am not sure this is a good name for the option. The files you are inputting are KDE files containing KDE values computed at the points of a template bank, they are not 'template files' as that is usually thought of in pycbc. I would call this eg `--kde-files`